src/components/selectable/selectable_list/selectable_list_item.tsx

```diff
@@ -105,48 +126,69 @@ export class EuiSelectableListItem extends Component<
     let instruction: React.ReactNode;
     if (allowExclusions && checked === 'on') {
       state = (
-        <EuiScreenReaderOnly>
-          <span>
-            <EuiI18n
-              token="euiSelectableListItem.includedOption"
-              default="Included option."
-            />
-          </span>
-        </EuiScreenReaderOnly>
+        <span>
+          <EuiI18n
+            token="euiSelectableListItem.includedOption"
+            default="Selected option."
+          />
+        </span>
       );
       instruction = (
-        <EuiScreenReaderOnly>
-          <span>
-            <EuiI18n
-              token="euiSelectableListItem.includedOptionInstructions"
-              default="To exclude this option, press enter."
-            />
-          </span>
-        </EuiScreenReaderOnly>
+        <span>
+          <EuiI18n
+            token="euiSelectableListItem.includedOptionInstructions"
+            default="To exclude this option, press enter."
+          />
+        </span>
       );
     } else if (allowExclusions && checked === 'off') {
       state = (
-        <EuiScreenReaderOnly>
-          <span>
-            <EuiI18n
-              token="euiSelectableListItem.excludedOption"
-              default="Excluded option."
-            />
-          </span>
-        </EuiScreenReaderOnly>
+        <span>
+          <EuiI18n
+            token="euiSelectableListItem.excludedOption"
+            default="Excluded option."
+          />
+        </span>
       );
       instruction = (
-        <EuiScreenReaderOnly>
-          <span>
-            <EuiI18n
-              token="euiSelectableListItem.excludedOptionInstructions"
-              default="To deselect this option, press enter."
-            />
-          </span>
-        </EuiScreenReaderOnly>
+        <span>
+          <EuiI18n
+            token="euiSelectableListItem.excludedOptionInstructions"
+            default="To uncheck this option, press enter."
+          />
+        </span>
+      );
+    } else if (allowExclusions && !checked) {
+      instruction = (
+        <span>
+          <EuiI18n
+            token="euiSelectableListItem.unckeckedOptionInstructions"
+            default="To select this option, press enter."
```

Comment:
I think we should include this text for non-exclusion EuiSelectables.

```suggestion
    } else if (!checked) {
      instruction = (
        <span>
          <EuiI18n
            token="euiSelectableListItem.unckeckedOptionInstructions"
            default="To select this option, press enter."
```

The basic example on https://eui.elastic.co/pr_5157/#/forms/selectable is super lacking on context clues and interactions compared to the exclusions example or the searchable example, and it's not clear to me why it should be so different and lacking in screen reader experience when it's way more likely that users will be using the basic functionality.